### PR TITLE
More supported variants of CLKey

### PR DESCRIPTION
**Pull request by hoffmannjan**
missing tests specified in [this ticket](https://github.com/casper-ecosystem/casper-js-sdk/issues/438)

### CODE REVIEW

File: src/lib/CLValue/KeyVariants.ts [hunk 1]
```diff
@@ -658,10 +567,91 @@ export class BidAddr implements CLKeyVariant {
...
+
+export class BidAddr implements CLKeyVariant {
+  keyVariant = KeyTag.BidAddr;
+  prefix = BID_ADDR_PREFIX;
+
+  bidAddrTag = BidAddrTag;
+
+  constructor(public data: BidAddrData) {}
+
+  value(): any {
```
Comment (ryoknz) [inline src/lib/CLValue/KeyVariants.ts]:
```suggestion
  value(): BidAddrData {
```

can we have type here?

File: src/lib/CLValue/KeyVariants.ts [hunk 2]
```diff
@@ -684,3 +674,33 @@ export class BidAddr implements CLKeyVariant {
...
   }
 }
+
+export class KeyPackage implements CLKeyVariant {
+  keyVariant = KeyTag.Package;
+  prefix = PACKAGE_PREFIX;
+
+  constructor(public data: Uint8Array) {}
+
+  value(): any {
```
Comment (ryoknz) [inline src/lib/CLValue/KeyVariants.ts]:
```suggestion
  value(): Uint8Array {
```

same here

**Review by ryoknz — APPROVED** [review]
Looks great, left some NITs
and please take a look failed test, https://github.com/casper-ecosystem/casper-js-sdk/actions/runs/9641365736/job/26586883018#step:8:19

File: src/lib/CLValue/Key.ts [hunk 3]
```diff
@@ -19,15 +15,47 @@ import {
...
+  BID_PREFIX,
+  WITHDRAW_PREFIX,
+  DICTIONARY_PREFIX,
+  SYSTEM_ENTITY_REGISTRY_PREFIX,
+  ERA_SUMMARY_PREFIX,
+  UNBOND_PREFIX,
+  CHAINSPEC_REGISTRY_PREFIX,
+  CHECKSUM_REGISTRY_PREFIX,
+  BidAddr,
+  BidAddrParser
```
Comment (zajko) [inline src/lib/CLValue/Key.ts:50]: Do we allow asterisk imports? Maybe it would be cleaner to just do `import * from './index'`?

**Review by zajko — APPROVED** [review]

File: src/lib/CLValue/Key.ts (hunk 3 above)
Comment (hoffmannjan) [reply src/lib/CLValue/Key.ts:50]: hm, do we use asterisk imports anywhere? its good idea but I would add this in 4.0.0 as standard

**Pull request merged**